packages/client/src/redux/slices/user-slice.ts

```diff
@@ -12,9 +12,18 @@ export type UserState = {
   username?: string;
   email?: string;
   reputation?: number;
-  // password?: string (shouldn't be stored on client)
   posts: Post[];
-  votedPosts: { [id: string]: { upvoted: boolean } }; // TODO idk if this is best way to do it
+  votedPosts: { [id: string]: { upvoted: boolean } };
```

Comment:
he decided it was the best way to do it

Reply:
This doesn't work for me when I first load the page. Can't we encode isUpvoted/isDownvoted into every post as it's retrieved?

Reply:
> he decided it was the best way to do it

narrator: it was not the best way to do it